src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -26,8 +26,10 @@
         d (Math/abs (- nref nalt))]
     (when (and (not= 1 nref) (not= 1 nalt)
                (some (fn [[s e]]
-                       (or (<= pos s (+ pos nref -1))
-                           (<= pos e (+ pos nref -1)))) exon-ranges))
+                       (and (not= s e)
```

Comment:
Usually `s` and `e` are different value.
But when calculate offset, `s` and `e` are same value and this sometimes cause `Variants overlapping a boundary of exon/intron are unsupported error`.
So I added `(not= s e)` condition.

https://github.com/chrovis/varity/blob/a155c3815c3fe7b54880c575228341ff35d09914/src/varity/vcf_to_hgvs/protein.clj#L93-L94